src/topology/metric_space/basic.lean

```diff
@@ -511,7 +512,13 @@ by rw [← ball_union_sphere, set.union_diff_cancel_right sphere_disjoint_ball.s
 by rw [← ball_union_sphere, set.union_diff_cancel_left sphere_disjoint_ball.symm]
 
 theorem mem_ball_comm : x ∈ ball y ε ↔ y ∈ ball x ε :=
-by simp [dist_comm]
+by {rw mem_ball', refl}
```

Comment:
I don't remember whether we have a coding style about spaces inside `{...}`. I would write it like
```suggestion
by { rw mem_ball', refl }
```
or
```suggestion
by rw [mem_ball', mem_ball]
```

Reply:
The second suggestion looks good to me - usually a single rewrite is preferrable to `refl`, as it's more helpful for a reader looking for API.

Reply:
Funnily enough that's actually what I originally had!

Reply:
Changed.